plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/PipelineStore.java

```diff
@@ -228,17 +275,28 @@ synchronized void updatePipelines() throws Exception {
         }
     }
 
-    void startUpdateWorker() {
-        threadPool.schedule(pipelineUpdateInterval, ThreadPool.Names.GENERIC, new Updater());
-    }
-
     boolean existPipeline(String pipelineId) {
```

Comment:
doesn't need to be part of this PR necessarily but maybe we should rename to pipelineExists ?